gradio/blocks.py

```diff
@@ -1047,3 +1047,19 @@ def block_thread(
             self.server.close()
             if self.enable_queue:
                 queueing.close()
+
+    def attach_load_events(self):
+        """Add a load event for every component whose initial value should be randomized."""
+        from gradio.components import IOComponent
```

Comment:
Can you move the import to the top of the file? I'm guessing you might be experiencing circular imports though. 

Did you try `from gradio import components` and then check `isinstance(component, components.IOComponent)`

Reply:
Awesome, yea importing `components` works so I'll push that up!